src/components/venn-diagram/VennDiagram.jsx

```diff
@@ -19,6 +19,7 @@ const VennDiagram = React.createClass({
     vennData: PropTypes.instanceOf(VennData).isRequired,
     setLabel: PropTypes.string,
     sizeLabel: PropTypes.string,
+    formatSize: PropTypes.func,
```

Comment:
I initially went with `sizeFormatter`, but `formatSize` is more in line with our "naming conventions". See the `renderTooltip()` prop in `HoverTooltip.jsx`.